app/src/main/java/msi/paria/trainingapp/pages/main/view/RegisterPageView.kt

```diff
@@ -0,0 +1,38 @@
+package msi.paria.trainingapp.pages.main.view
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import msi.paria.trainingapp.widgets.buttons.RoundButtonView
+import msi.paria.trainingapp.widgets.textFileds.EmailTextFiledView
+
+
+@Composable
+fun RegisterPageView(
+    emailTextFiledId: String, userEmail: String, emailValidateState: Boolean, onEmailChanged: (newEmail: String) -> Unit,
+    buttonFiledId: String, buttonText: String, onSubmitButtonClick: () -> Unit
+) {
+
+    if (emailTextFiledId == buttonFiledId)
```

Comment:
What did you add this code for?

Reply:
Same id not allowed in single page. this line prevent to use same id for more then one view.

Reply:
it's not necessary to check this item

Reply:
it's not necessary for showing view, but when you want to find single node with tag, your app crashed because it's find more than one item.